src/components/home/elements/Record.vue

```diff
@@ -127,6 +128,156 @@
                  v-if="transType==='lease'">
     </CancelLease>
   </b-container>
```

Comment:
The code added below is almost the same as the above. I do not suggest doing like this. For send to self case, I suggest change getTxRecords() method in `Records.vue`. We could duplicate the record in once get http response and set recipient to "self". This is a little bit trick but use much less code and easy to maintain.